### PR TITLE
Add point to TOC / fix typo

### PR DESCRIPTION
**Pull request by ggruening**
This one cleans up https://github.com/kereis/traefik-certs-dumper/pull/82 a little bit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ Special thanks to them!
   - [Health Check](#health-check)
   - [Merging private key and public certificate in one .pem](#merging-private-key-and-public-certificate-in-one-pem)
   - [Merging private key and public certificate in one PKCS12 file](#merging-private-key-and-public-certificate-in-one-pkcs12-file)
+  - [Convert Keys in RSA Format](#convert-keys-in-rsa-format)
```
Comment (kereis) [inline README.md:45]:
```suggestion
  - [Convert Keys in RSA format](#convert-keys-in-rsa-format)
```

Tiny typo fix. :)

File: README.md [hunk 2]
```diff
@@ -279,7 +280,7 @@ secrets:
 
 ### Convert keys in RSA format
 
-Some applications like [mySQL](https://www.mysql.com/) or [mariaDB](https://mariadb.org/) need their keys in RSA format. In this case, you can set the environment variable `CONVERT_KEYS_TO_RSA`. Each time `traefik-certs-dumper` dumps the certificates, this script will create a file named `rsakey.pem` in each domain's folder respectively. If required, this file name can be configured using the environment variables `RSA_KEY_FILE_NAME` und `RSA_KEY_FILE_EXT`.
+Some applications like [mySQL](https://www.mysql.com/) or [mariaDB](https://mariadb.org/) need their keys in RSA format. In this case, you can set the environment variable `CONVERT_KEYS_TO_RSA`. Each time `traefik-certs-dumper` dumps the certificates, this script will create a file named `rsakey.pem` in each domain's folder respectively. If required, this file name can be configured using the environment variables `RSA_KEY_FILE_NAME` and `RSA_KEY_FILE_EXT`.
```
Comment (kereis) [inline README.md:283]:
```suggestion
Some applications like [MySQL](https://www.mysql.com/) or [mariaDB](https://mariadb.org/) need their keys in RSA format. In this case, you can set the environment variable `CONVERT_KEYS_TO_RSA`. Each time `traefik-certs-dumper` dumps the certificates, this script will create a file named `rsakey.pem` in each domain's folder respectively. If required, this file name can be configured using the environment variables `RSA_KEY_FILE_NAME` and `RSA_KEY_FILE_EXT`.
```

Also tiny typo fix.

**Review by kereis — CHANGES_REQUESTED** [review]
Ah thank you. 😄 Didn't pay attention to that. 

Got tiny suggestions for you.

**Comment by ggruening** [comment]
> Didn't pay attention to that.

Same here 😉 Maybe it's done now. 🙂

**Review by kereis — APPROVED** [review]
Let's these were the only typos for now. :D Thanks!

**Pull request merged**